.github/workflows/shellcheck.yml

```diff
@@ -0,0 +1,19 @@
+name: Shellcheck
+
+on:
+  push:
+    branches:
+      - 'master'
+      - 'release-**'
+  pull_request:
+
+jobs:
+
+  shellcheck:
+    name: Shellcheck
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
```

Comment:
would it take that long to complete? I saw the only 2 runs last month failed in less than a minute. Will the complete run take roughly the same time to complete? If so, I think we should remove this line, otherwise, it kind of suggests this check could last pretty long when in reality it isn't the case.
<img width="1057" alt="image" src="https://user-images.githubusercontent.com/1937582/226903069-aa0fabec-34d1-41d1-a26e-47a4bf768d42.png">

Reply:
Good point. For some reason, I thought the default was a minute or two.